src/coreclr/src/md/compiler/mdperf.cpp

```diff
@@ -1,95 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-//*****************************************************************************
-// MDperf.cpp
-//
-
-//
-// This file provides Compiler Support functionality in metadata.
-//*****************************************************************************
-
-#include "stdafx.h"
-
-#include "mdperf.h"
```

Comment:
mdperf.h should be deleted too.

Reply:
Reverted this file as per the other comment.